extensions/wikia/Wall/WallBaseController.class.php

```diff
@@ -70,7 +70,9 @@ public function thread() {
 		if ( !empty( $title ) && $title->exists() && in_array( MWNamespace::getSubject( $title->getNamespace() ), $this->app->wg->WallNS ) ) {
 			$wallMessage = WallMessage::newFromTitle( $title );
 			$wallMessage->load();
-			$this->app->wg->Out->setPageTitle( $wallMessage->getMetaTitle() );
+			// VOLDEV-3: fix Thread watchlist issues
+			$this->wg->User->clearNotification( $title );
+			$this->wg->Out->setPageTitle( $wallMessage->getMetaTitle() );
 		}
 
 		// TODO: keep the varnish cache and do purging on post
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)